src/test/java/com/productdock/library/rental/service/RentalRecordApiTest.java

```diff
@@ -119,20 +154,24 @@ void shouldReturnBadRequest_whenReservingABookAlreadyReservedByUser() throws Exc
 
     @Test
     void shouldReturnBadRequest_whenReturningABookNotRentedByUser() throws Exception {
-        makeRentalRequest(RentalStatus.RETURNED)
+        makeRentalRequestWithDefaultUser(RentalStatus.RETURNED)
                 .andExpect(status().isBadRequest());
     }
 
-    private ResultActions makeRentalRequest(RentalStatus request) throws Exception {
+    private ResultActions makeRentalRequest(RentalStatus request, String email) throws Exception {
         return mockMvc.perform(post("/api/rental/record")
                         .contentType(MediaType.APPLICATION_JSON)
                         .content("{\n" +
                                 "    \"bookId\": \"" + FIRST_BOOK + "\",\n" +
                                 "    \"requestedStatus\": \"" + request + "\"\n" +
-                                "}").with(jwt().jwt(jwt -> jwt.claim("email", PATRON))))
+                                "}").with(jwt().jwt(jwt -> jwt.claim("email", email))))
                 .andDo(print());
     }
 
+    private ResultActions makeRentalRequestWithDefaultUser(RentalStatus request) throws Exception {
```

Comment:
This can also be called makeRentalRequest, we don't want to pay attention to the user in this scenario, so hide it from the name too.